src/components/ScholarshipList.jsx

```diff
@@ -39,6 +40,15 @@ const useStyles = makeStyles((theme) => ({
 
 function ScholarshipList({ scholarships }) {
   const classes = useStyles();
+  const [shareDialogOpen, setShareDialogOpen] = useState(false);
+  const showShareDialog = () => setShareDialogOpen(true);
+  const closeShareDialog = () => setShareDialogOpen(false);
+
+  const [shareSiteLink, setShareSiteLink] = React.useState('');
+  const handleURLClick = (event) => setShareSiteLink(event);
+  const [shareSiteTitle, setShareSiteTitle] = React.useState('');
+  const handleTitleClick = (event) => setShareSiteTitle(event);
+
   const shareFn = (id, data) => () => {
     const title = `${data.amount} - ${data.name} | ${BRAND_NAME}`;
     const url = `${window.location.hostname}/scholarships/${id}`;
```

Comment:
I forgot to do this when I initially wrote this logic but could you prepend `https://` to the URL?

Reply:
Ok yeah ill add it